Cargo.toml

```diff
@@ -29,3 +31,8 @@ tempfile = "3.3"
 [dev-dependencies]
 assert_cmd = "2.0"
 predicates = "2.1"
+
+[profile.release] 
+strip = true  # Automatically strip symbols from the binary. 
+# opt-level = "z"  # Optimize for size. 
```

Comment:
You can also pick optimize for speed (there are 3 levels) with
opt-level = 3

Reply:
Yes, I just haven't experimented enough with the different levels. I know that z made working with large files considerably slower, so I left that out.

Reply:
2 & 3 had similar performance.  1 was noticeably slower.

Reply:
Ok, I'll take your word for it